### PR TITLE
deploy scripts support Conjur v5 enterprise

### LINKED ISSUE #16: deploy scripts support Conjur v5 enterprise
Update the deploy scripts to work with the Conjur v5 enterprise appliance.

### PR DESCRIPTION
**Pull request by jtuttle**
Closes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,19 @@ environment variables. The setup instructions below walk you through the
 necessary steps for configuring your environment and show you which variables
 need to be set before deploying.
 
+### Conjur Version
+
+If you are workign with Conjur v4, you will need to set:
```
Comment (dustinmm80) [inline README.md]: little typo here: workign

File: kubernetes/conjur-cluster.yaml [hunk 2]
```diff
@@ -22,6 +22,10 @@ spec:
       - name: conjur-appliance
         image: {{ CONJUR_APPLIANCE_IMAGE }}
         imagePullPolicy: Always
+        env:
+          # Only required for Conjur v5
+          - name: CONJUR_DATA_KEY
+            value: "{{ DATA_KEY }}"
```
Comment (dustinmm80) [inline kubernetes/conjur-cluster.yaml]: any reason not to call this var `CONJUR_DATA_KEY` as well? more explicit when you see it in scripts, i think

File: README.md (hunk 1 above)
Comment (jtuttle) [reply README.md]: Fixed.

File: kubernetes/conjur-cluster.yaml (hunk 2 above)
Comment (jtuttle) [reply kubernetes/conjur-cluster.yaml]: Nope, that works. Will change it.

**Comment by dustinmm80** [comment]
We need a few more things before we can merge:

- [ ] default `CONJUR_VERSION` to `CONJUR_MAJOR_VERSION`, since this is what summon-conjur uses
- [ ] fix Jenkinsfile conflict in this PR
- [ ] fix Jenkins build

Once these are done we're all set to squash/merge.

File: 8_print_config.sh [hunk 3]
```diff
@@ -6,10 +6,16 @@ set -euo pipefail
 set_namespace $CONJUR_NAMESPACE_NAME
 
 if [ $PLATFORM = 'kubernetes' ]; then
-    ui_url="https://$(get_master_service_ip):443"
+  ui_url="https://$(get_master_service_ip):443"
```
Comment (dustinmm80) [inline 8_print_config.sh]: any reason to add 443 on the end here? if it's https:// prefix it's 443 by default. Looks a bit weird in output.

Comment (jtuttle) [reply 8_print_config.sh]: Nope, will fix.

**Pull request merged**